frontend/packages/ceph-storage-plugin/src/components/kms-config/vault-config.tsx

```diff
@@ -64,6 +64,7 @@ export const ValutConfigure: React.FC<KMSConfigureProps> = ({
   };
 
   const setAuthMethod = (authMethod: VaultAuthMethods) => {
+    vaultStateClone.authValue.value = '';
```

Comment:
I dont think the user will switch to a different auth method once he enters details. That is why auth method dropdown is on top, they will decide which auth method to use then they will fill out the form.

I dont think this change is necessary.

Reply:
I think ask for the BZ was to reset entire state when auth method is changed not just resetting the authValue.

Reply:
@GowthamShanmugam do you want to close this BZ ?

Reply:
I dont find any issue with the old value displayed when auth method switches, most of the fields are common and it reduces user effort.  i dont think its a bug.

Reply:
okay then let's talk about authValue... it is not common between two. Shouldn't we at least reset that (will reduce the scope of the BZ from resetting entire state to just value of role/token) ?

Reply:
mm ya it can be reset, i agree

Reply:
okay then this PR looks good... I will comment on the BZ about this as well...

Reply:
ack

Reply:
`setAuthValue('')`